src/tests/base/test_orders.py

```diff
@@ -1759,11 +1759,19 @@ def test_cancel_and_change_addon(self):
 
     @classscope(attr='o')
     def test_cancel_all_in_order(self):
+        self.shirt.category = self.event.categories.create(name='Add-ons', is_addon=True)
+        self.ticket.addons.create(addon_category=self.shirt.category)
+        self.ocm.add_position(self.shirt, None, Decimal('13.00'), self.op1)
```

Comment:
```suggestion
        self.ocm.add_position(self.shirt, None, Decimal('13.00'), addon_to=self.op1)
```
(slightly more readable)